spec.html

```diff
@@ -32848,7 +32913,7 @@ <h1>RegExp.prototype [ @@replace ] ( _string_, _replaceValue_ )</h1>
           1. If Type(_rx_) is not Object, throw a *TypeError* exception.
           1. Let _S_ be ? ToString(_string_).
           1. Let _lengthS_ be the number of code unit elements in _S_.
-          1. Let _functionalReplace_ be IsCallable(_replaceValue_).
+          1. Let _functionalReplace_ be ! IsCallable(_replaceValue_).
```

Comment:
I prefer this left as is, without `!`.

Reply:
Until we have #1796, most every abstract operation should be using the prefix. Why would it not need the prefix?

Reply:
i suppose we do only have 5 of them prefixed at this point; although that's just an oversight from previous editorial direction, we can skip the prefix here for now